nco/readme.md

```diff
@@ -0,0 +1,7 @@
+# Repository for redirecting to Natural Conversational Ontology (NCO)
+
+# NCO artifact
+- http://w3id.org/nco --> https://raw.githubusercontent.com/ProfTuan/NCO/refs/heads/main/nco.owl
+
+# Maintainers/Contact
+- [Tuan Amith] (http://github.com/ProfTuan) muamith AT utmb DOT edu
```

Comment:
Looks like this was supposed to be a markdown link but had an extra space:
```suggestion
- [Tuan Amith](http://github.com/ProfTuan) muamith AT utmb DOT edu
```

Reply:
thanks! for the fix